<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="OptionsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="StrategyWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>701</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SymbolLe">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SymbolTl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshPb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastUpdateLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastPriceLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BidLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImpPctLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DividendPctLe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LastUpdateTL">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="BidAskTL">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bid / Ask</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImpHisPctTL">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imp / His %</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastPriceTL">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="DividendPctTL">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dividend %</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AskLe">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="HisPctLe">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutPB">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
